tests/unit/test_models_rates.py

```diff
@@ -202,7 +202,6 @@ def test_rates_method_to_embed(sample_rates_dict):
 3 × Hatching
 0.6 × Cuddle Interval
 3 × Imprinting
-1.5 × Hexagon Reward
-69420 × MyMadeUpValue\
+1.5 × Hexagon Reward\
```

Comment:
I believe this should confirm whether items are correctly stripped. 
I looked through the other tests and don't _think_ they should be effected by this change, but we'll see.